babel.config.js

```diff
@@ -7,7 +7,6 @@ module.exports = function( api ) {
 		],
 		plugins: [
 			/** this is needed because plugin exit modal uses react not wp.element */
-			'@babel/transform-react-jsx',
```

Comment:
what about the comment right above this line?

> this is needed because plugin exit modal uses react not wp.element

Reply:
oh ya, the comment could have been removed too!  Dunno how I missed that.

Reply:
Fixed in ffb080c